File: tests/patterns/java/import_metavar_fullpath.java
// what actually matters in this test is the value bound to $LIB
// in the .sgrep. It should be the full path!
// use semgrep-core ... -pvar '$X' to print the value, or -json

  
//ERROR: match
import org.bouncycastle.crypto.provider.BouncyCastleProvider;

File: tests/patterns/python/import_metavar_fullpath.py
# what actually matters in this test is the value bound to $X
# in the .sgrep. It should be the full path!
# use semgrep-core ... -pvar '$X' to print the value.

#ERROR: match
import a
#ERROR: match
import auth
#ERROR: match
import arbitrary_auth
#ERROR: match
import a.auth
#ERROR: match
import b.auth
#ERROR: match
import a.b.auth
